src/xpath.c

```diff
@@ -5493,15 +5493,17 @@ moveto_snode_root(struct lyxp_set *set, struct lys_node *cur_node, int options)
  *
  * @param[in] node Node to check.
  * @param[in] root_type XPath root node type.
- * @param[in] node_name Node name to move to. Must be in the dictionary!
+ * @param[in] node_name_dict Contains the node name to move to, when it is on the dict. NULL otherwise.
```

Comment:
Doc should probably say that it is an optional name in the dictionary. What you wrote I find a bit confusing.